### PR TITLE
Editing item new styles

### PR DESCRIPTION
**Pull request by sanredd**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ yarn change`

#### Description of changes

Added style and types file for EditingItem component.
Adding semanticColors for theme support.

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by sanredd** [comment]
@micahgodbolt Please review.
Just component cleanup.

**Comment by sanredd** [comment]
@joschect please review this PR

File: packages/office-ui-fabric-react/src/components/SelectedItemsList/SelectedPeopleList/Items/EditingItem.styles.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { getGlobalClassNames, getTheme } from '@uifabric/styling';
```
Comment (aneeshack4) [inline packages/office-ui-fabric-react/src/components/SelectedItemsList/SelectedPeopleList/Items/EditingItem.styles.ts]: Where is this component on the PR deploy? Or if it's not in the generic components list, where can I find its usage?

Comment (sanredd) [reply packages/office-ui-fabric-react/src/components/SelectedItemsList/SelectedPeopleList/Items/EditingItem.styles.ts]: This is not a generic component.
This component is used in SelectedPeopleList.

Comment (aneeshack4) [reply packages/office-ui-fabric-react/src/components/SelectedItemsList/SelectedPeopleList/Items/EditingItem.styles.ts]: What is SelectedPeopleList? Can you point me to a link of the docs for this component?

Comment (sanredd) [reply packages/office-ui-fabric-react/src/components/SelectedItemsList/SelectedPeopleList/Items/EditingItem.styles.ts]: https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/SelectedItemsList/SelectedPeopleList/SelectedPeopleList.tsx
Above is the component where we are using the EditingItem line number 88.

**Pull request merged**